pkgs/build-support/cc-wrapper/cc-wrapper.sh

```diff
@@ -246,10 +246,13 @@ if [[ -e @out@/nix-support/cc-wrapper-hook ]]; then
 fi
 
 if (( "${NIX_CC_USE_RESPONSE_FILE:-@use_response_file_by_default@}" >= 1 )); then
-    exec @prog@ @<(printf "%q\n" \
+    responseFile=$(mktemp --tmpdir cc-params.XXXXXXXXXX)
```

Comment:
```suggestion
    responseFile=$(mktemp --tmpdir cc-params.XXXXXX)
```
https://github.com/NixOS/nixpkgs/pull/178626